SimTracker/SiPixelDigitizer/plugins/SiPixelChargeReweightingAlgorithm.h

```diff
@@ -0,0 +1,88 @@
+#ifndef SimTracker_SiPixelDigitizer_SiPixelChargeReweightingAlgorithm_h
+#define SimTracker_SiPixelDigitizer_SiPixelChargeReweightingAlgorithm_h
+
+#include "SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.h"
+
+typedef boost::multi_array<float, 2> array_2d;
```

Comment:
is this needed as it's already defined at line 63?

Reply:
Line 6 is not needed --> removed.
But line 4 is really needed because it is used for 
 typedef std::map<int, SiPixelDigitizerAlgorithm::Amplitude, std::less<int> > signal_map_type;
on new line 42.